lib/signet/test/signer.ex

```diff
@@ -21,8 +21,6 @@ defmodule Signet.Test.Signer do
       name: name
     )
 
-    GenServer.cast(name, :set_address)
```

Comment:
this means the transparent flow is implicitly tested many times over, but an explicit test would be nice...

Reply:
Yeah, this gets tested a ton, and I'm glad we don't need this anymore, as it was a hack.